doc/syscall-interception.md

```diff
@@ -37,6 +37,52 @@ at all.
 
 This can be enabled by setting `security.syscalls.intercept.mknod` to `true`.
 
+### bpf
+The `bpf` system call is used to manage eBPF programs in the kernel.
+Those can be attached to a variety of kernel subsystems.
+
+In general, loading of untrusted eBPF program can be problematic as it
+can facilitate timing based attacks.
+
+LXD's eBPF support is currently restricted to allowing programs use to
```

Comment:
It feels like this whole section should indicate that these settings are applied to instances rather than at a server level, as it doesn't indicate currently where these settings should be set.